Add missing action_limit settings and new policy_limit settings (#780)
elastic/ingest-docs: docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc

`server.limits`::
`policy_throttle`:::

> How often a new policy is rolled out to the agents.
> Deprecated: Please use policy_limits settings.

Reviewer: ```suggestion
How often a new policy is rolled out to the agents.
+
Deprecated: Use the `policy_limit` settings instead.
```

`server.limits`::
`policy_throttle`:::
How often a new policy is rolled out to the agents.
+
Deprecated: Use the `policy_limit` settings instead.

`action_limit.interval`:::

> How fast the fleet-server dispatches pending actions to the agents.

Reviewer: ```suggestion
How quickly {fleet-server} dispatches pending actions to the agents.
```

`action_limit.burst`:::

> Burst of actions that may be dispatched befoe falling back to the rate limit defined by `interval`.

Reviewer: ```suggestion
Burst of actions that may be dispatched before falling back to the rate limit defined by `interval`.
```

`policy_limit.interval`:::

> How fast the fleet-server dispatches pending policies to the agents.

Reviewer: ```suggestion
How quickly {fleet-server} dispatches pending policies to the agents.
```
